typings/custom-oppia-defs.d.ts

```diff
@@ -0,0 +1,22 @@
+// These declarations are required to avoid compile errors while compiling
```

Comment:
We should talk about this, I don't know what the criterion is for files to go in here, and this means it cannot be enforced.

Reply:
This will be removed by using typescript in watch mode.